zerver/openapi/zulip.yaml

```diff
@@ -416,6 +416,7 @@ paths:
                                             The ID of the user who is affected by this change.
                                         avatar_url:
                                           type: string
+                                          nullable: true
```

Comment:
You need to add `**Changes**` entry here and also the appropriate changes in `changelog.md` and `version.py`.